clusterloader2/pkg/prometheus/manifests/exporters/kube-state-metrics/deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: kube-state-metrics
+    app.kubernetes.io/version: 2.0.0-rc.0
+  name: kube-state-metrics
+  namespace: kube-state-metrics-perf-test
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kube-state-metrics
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: kube-state-metrics
+        app.kubernetes.io/version: 2.0.0-rc.0
+    spec:
```

Comment:
we definitely need to set requests/limits (otherwise this can be easily starved by the tests themselves)

Reply:
Makes sense, its a chicken and egg problem I guess. We want to use these tests to suggest requests and limits to users but need to set some as well I understand. 

I took the example from the Prometheus you have deployed but also added CPU to the limits so we have a guaranteed QoS.